src/core/DOMEval.js

```diff
@@ -3,12 +3,19 @@ define( [
 ], function( document ) {
 	"use strict";
 
-	function DOMEval( code, doc ) {
+	function DOMEval( code, doc, type, src ) {
```

Comment:
See https://github.com/jquery/jquery/pull/3766. I think this is sufficient cause to pass around the node object itself, instead of adding parameter after parameter.

Reply:
function `globalEval` depends from `DOMEval`, probably we should write another function something like `DOMScript`, that function can receive node object. also this function can include all checks like 
```javascript
	if ( rscriptType.test( node.type || "" ) &&
		!dataPriv.access( node, "globalEval" ) &&
		jQuery.contains( doc, node ) ) {

		if ( node.src && node.type !== "module" ) {
```

Reply:
`DOMEval` is already an internal function, and only invoked at two preexisting sites. We don't need to introduce another function, we just need to update the signature to e.g. `function DOMEval( code, doc, node )` and copy over whitelisted attributes (`src`, `type`, `crossorigin`, `integrity`, etc.) in the body.